internal/pkg/db/postgres/utils.go

```diff
@@ -144,3 +144,8 @@ func WrapDBError(message string, err error) errors.EdgeX {
 	}
 	return errors.NewCommonEdgeX(errors.KindDatabaseError, message, err)
 }
+
+// NormalizePath normalizes file paths to be platform-independent.
+func NormalizePath(baseDir, fileName string) string {
+    return filepath.ToSlash(filepath.Join(baseDir, fileName))
```

Comment:
according to the go docs, filepath.Join should use the OS separator.
https://pkg.go.dev/path/filepath#Join

Windows is `\` and Linux is `/`.  If you wrap it with filepath.ToSlash, will it become always `/`?
Could this really solve the issue?